cypress/e2e/API/EngageSphere.cy.js

```diff
@@ -0,0 +1,179 @@
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+
+describe('Teste de API - Filtragem de clientes por tamanho', () => {
+  const sizes = ['Small', 'Medium', 'Enterprise', 'Large Enterprise', 'Very Large Enterprise'];
+
+  sizes.forEach((size) => {
+    it(`Deve retornar apenas clientes com o tamanho "${size}"`, () => {
+      cy.request('GET', `${CUSTOMERS_API_URL}?page=1&limit=50&size=${size}`)
+        .then(({ status, body }) => {
+          expect(status).to.equal(200);
+          expect(body).to.have.property('customers');
+
+          const clients = body.customers;
+          expect(clients).to.be.an('array').not.empty;
```

Comment:
E então aqui, fazer o seguinte:

```suggestion
          expect(customers).to.be.an('array').not.empty;
```